packages/react-portal-compat/.babelrc.json

```diff
@@ -1,4 +1,3 @@
 {
-  "presets": ["@griffel"],
```

Comment:
We don't use Griffel these, so I decided to remove it. I can restore for consistency if needed.